ignite/config/chain/v0/config.go

```diff
@@ -6,20 +6,20 @@ import (
 	"github.com/imdario/mergo"
 	"gopkg.in/yaml.v2"
 
-	"github.com/ignite/cli/ignite/chainconfig/config"
+	"github.com/ignite/cli/ignite/config/chain/base"
 )
 
 // Config is the user given configuration to do additional setup during serve.
 type Config struct {
-	config.BaseConfig `yaml:",inline"`
+	base.Config `yaml:",inline"`
 
-	Validator Validator   `yaml:"validator"`
-	Init      config.Init `yaml:"init"`
-	Host      config.Host `yaml:"host"`
+	Validator Validator `yaml:"validator"`
+	Init      base.Init `yaml:"init"`
+	Host      base.Host `yaml:"host"`
 }
 
 // Clone returns an identical copy of the instance.
-func (c *Config) Clone() (config.Converter, error) {
+func (c *Config) Clone() (base.Converter, error) {
```

Comment:
It might be a good idea to rename `Converter` to `ConfigConverter`

Reply:
[[360bc17](https://github.com/ignite/cli/pull/3182/commits/360bc17d3aa74645f195a97e87837e510e88de45)](https://github.com/ignite/cli/pull/3182/commits/360bc17d3aa74645f195a97e87837e510e88de45) should make things a bit clearer